<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-31T17:28:38 IST" name="Default suite" finished-at="2025-01-31T17:29:29 IST" duration-ms="51329">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:stepdefinitions.TestRunner@175c2241]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2025-01-31T17:28:38 IST" name="Default test" finished-at="2025-01-31T17:29:29 IST" duration-ms="51329">
      <class name="stepdefinitions.TestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:stepdefinitions.TestRunner@175c2241]" started-at="2025-01-31T17:28:38 IST" name="setUpClass" finished-at="2025-01-31T17:28:38 IST" duration-ms="239" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@1532c619]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:stepdefinitions.TestRunner@175c2241]" started-at="2025-01-31T17:28:38 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2025-01-31T17:29:29 IST" duration-ms="51048" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Check login is successfull with valid credentails"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["login functionality"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[verified expected [https://facebook.com] but found [https://www.facebook.com/login/?privacy_mutation_token=eyJ0eXBlIjowLCJjcmVhdGlvbl90aW1lIjoxNzM4MzI0NzIxLCJjYWxsc2l0ZV9pZCI6MzgxMjI5MDc5NTc1OTQ2fQ%3D%3D&next]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: verified expected [https://facebook.com] but found [https://www.facebook.com/login/?privacy_mutation_token=eyJ0eXBlIjowLCJjcmVhdGlvbl90aW1lIjoxNzM4MzI0NzIxLCJjYWxsc2l0ZV9pZCI6MzgxMjI5MDc5NTc1OTQ2fQ%3D%3D&next]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at stepdefinitions.runclass.the_user_should_be_directed_to_home_page(runclass.java:41)
at âœ½.the user should be directed to home page(file:///C:/Users/Sidharth%20Radeep/eclipse-workspace/Cucumber_project/src/test/resources/Features/login1.feature:8)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:stepdefinitions.TestRunner@175c2241]" started-at="2025-01-31T17:29:29 IST" name="tearDownClass" finished-at="2025-01-31T17:29:29 IST" duration-ms="14" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- stepdefinitions.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
